### DISCUSSION TITLE
FSL intensity normalization or grand mean scaling?

### DISCUSSION THREAD

**Discussion by lexyganesh** [maartenmennes/ICA-AROMA#77] [Q&A]
Hi! I have a clarification question about the fMRI preprocessing from the paper to prep for ICA-AROMA. From the text: _"Preprocessing of the training set was carried out using tools from the FMRIB Software Library and involved (1) removal of the first five volumes to allow for signal equilibration, (2) head movement correction by volume-realignment to the middle volume using MCFLIRT, (3) global 4D mean intensity normalization and (4) spatial smoothing (6 mm FWHM). Importantly, no temporal filtering was applied at this stage of processing...."_ For the step 3 "global 4D mean intensity normalization" did you use FSL's default grand mean scaling or the FSL option for intensity normalization? The note of 4D makes me think the former (since intensity normalization is usually 3D), but I had not heard grand mean scaling referred to as intensity normalization before and it was listed like an option rather than a default, so I just want to be sure. Thank you!!

**Comment by maartenmennes** [comment]
it's the default grand mean scaling, NOT the 'intensity normalization' option in FEAT. I agree the terminology is confusing...
